test/test.js

```diff
@@ -112,22 +112,22 @@ function createTest(rawSubTest) {
     // eslint-disable-next-line no-param-reassign
     t.context.cleanup = rawSubTest.cleanup;
 
-    const matchText = function matchText(expected, text) {
-      return createRegTest(expected).test(text);
+    const matchText = function matchText(expected, text, prop) {
+      return createRegTest(expected[prop]).test(text[prop]);
     };
 
     for (const step of rawSubTest.steps) {
       // eslint-disable-next-line no-await-in-loop
       const cmdOut = await t.context.ct.streamIn(step.in);
       if (step.out) {
-        const stripStdout = stripText(cmdOut.stdout);
-        t.true(matchText(step.out, stripStdout));
+        const stripCmdOut = { ...cmdOut, out: stripText(cmdOut.stdout) };
```

Comment:
Nit: this means that the power-assert will have both `stdout` and `out`, which are almost the same, except for whitespace.

Reply:
Changed it, `matchText` is a little weird now but should work
Will make https://ci.binaris.io/blue/organizations/jenkins/gobinaris%2Fprecious/detail/master/7110/pipeline fail again to see the errors